### PR TITLE
[bitnami/metrics-server] Allow adding side containers to the pod

### PR DESCRIPTION
**Pull request by shulima**
**Description of the change**

Add an `extraContainers` option, which allows running side containers within the pod by defining them in values.yaml.

**Benefits**

This allows running extra utility containers, like addon-resizer, alongside metrics-server.

**Possible drawbacks**

Can't think of anything.

**Applicable issues**

None.

**Additional information**

This is something we need for our clusters, and while we could simply fork, we think others might benefit from the option to run side containers.

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

File: bitnami/metrics-server/templates/deployment.yaml [hunk 1]
```diff
@@ -49,6 +49,9 @@ spec:
       hostNetwork: true
       {{- end }}
       containers:
+        {{- if .Values.extraContainers }}
+        {{- toYaml .Values.extraContainers | nindent 8 }}
```
Comment (Mauraza) [inline bitnami/metrics-server/templates/deployment.yaml:53]: In other charts we have this:
```yaml
{{- include "common.tplvalues.render" (dict "value" .Values.extraContainers "context" $) | nindent 8 }}
```
Could you change it?

**Review by Mauraza — COMMENTED** [review]
Hi @shulima,

Thanks for the PR!, could you check the comment?

File: bitnami/metrics-server/templates/deployment.yaml (hunk 1 above)
Comment (shulima) [reply bitnami/metrics-server/templates/deployment.yaml]: @Mauraza Change applied, thanks.

File: bitnami/metrics-server/Chart.yaml [hunk 2]
```diff
@@ -23,4 +23,5 @@ name: metrics-server
 sources:
   - https://github.com/bitnami/bitnami-docker-metrics-server
   - https://github.com/kubernetes-incubator/metrics-server
-version: 5.8.13
+version: 5.8.14
+
```
Comment (Mauraza) [inline bitnami/metrics-server/Chart.yaml:27]: Could you remove the black line?

**Review by Mauraza — COMMENTED** [review]
Hi @shulima,

Could you check the lint errors?

File: bitnami/metrics-server/Chart.yaml (hunk 2 above)
Comment (shulima) [reply bitnami/metrics-server/Chart.yaml]: Removed. That's what I get for resolving conflicts through the web interface 😔

**Comment by Mauraza** [comment]
Hi @shulima,

Thanks for the PR!  LGTM!!

**Pull request merged**